scripts/run-postInstall.js

```diff
@@ -0,0 +1,12 @@
+const os = require('os')
+const { execSync } = require('child_process')
+
+if (os.platform() === 'win32' && !process.env.CI) {
```

Comment:
Excluding CI for this caused the `postinstall` step in the `windows-node-modules-install` job to fail on Windows in CI due to the non-windows version of this code running. Can we remove the CI check safely, or do we need another condition for windows on CI?

Here is a copy [run based on a copy of this branch with windows tests enabled](https://app.circleci.com/pipelines/github/cypress-io/cypress/35384/workflows/5275720e-e097-4a07-a8af-860b05258000/jobs/1408820) in CI. We see this error:

```
Error: Command failed: patch-package && ./scripts/run-if-not-ci.sh yarn-deduplicate --strategy=highest && yarn clean && gulp postinstall && ./scripts/run-if-not-ci.sh yarn build
```